burn-wgpu/src/lib.rs

```diff
@@ -20,7 +20,11 @@ pub use graphics::*;
 
 #[cfg(test)]
 mod tests {
-    type TestBackend = crate::WGPUBackend<crate::Vulkan, f32, i64>;
+    use super::*;
+
+    type TestBackend = WGPUBackend<Vulkan, f32, i64>;
```

Comment:
Would it possible to use CPU or virtual device to test the WGPU backened? It'd be great if we could run the tests locally or on CI.

Reply:
It is running with the CPU device, but using the Vulkan Graphics API :)